community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf

```diff
@@ -95,6 +97,29 @@ locals {
 
     # Providers
     endpoint_versions = var.endpoint_versions
+
+    # Extra-files MD5 hashes
+    # Makes config file creation depend on the files
+    # Allows for informed updates & checks on slurmsync side
+    slurm_gcp_scripts_md5 = google_storage_bucket_object.devel.md5hash,
```

Comment:
Just confirming that these `*_md5` are to-be-used by https://github.com/GoogleCloudPlatform/slurm-gcp at some later point in time?

Reply:
yes, it's not explicitly used now, implicitly it's used for temporal ordering.